From c640cb2c58fa92ba108deab41c19e6db6c6270d4 Mon Sep 17 00:00:00 2001
From: "Suresh, Prashanth" <prashanth.suresh@intel.com>
Date: Mon, 26 Sep 2022 15:36:11 +0530
Subject: [PATCH] Pass host graphics SRIOV info to guest.

Add file in aaf folder to enable guest to know if host supports SRIOV.
Apply patch in setup_host.sh for vm-manager.

Tracked-On: OAM-104076
---
 scripts/setup_host.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
index 79450d6..fbfc44f 100755
--- a/scripts/setup_host.sh
+++ b/scripts/setup_host.sh
@@ -159,6 +159,7 @@ function install_vm_manager_src() {
         git clone https://github.com/projectceladon/vm_manager.git || return -1
     fi
     cd vm_manager/
+    git apply $CIV_VERTICAl_DIR/vm-manager/*.patch
     make || return -1
     sudo make install || return -1
     cd -
-- 
2.37.3

